docs/opentracing.rst

```diff
@@ -92,6 +92,26 @@ two problems, namely:
   but that doesn't prevent another server sending you baggage which will be logged
   to OpenTracing's logs.
 
+==========
+EDU FORMAT
+==========
+
+EDUs can contain tracing data in their content. This is not specced but
+it could be of interest for other homeservers.
+
+Edu format:
+
+.. code-block:: json
+
+   edu: {
+     edu_type: "type"
+     content: {
+       context: {
+         ...opentracing_context
```

Comment:
An example of how it's used or what it contains? I can't make an example of the content because that's completely vendor specific and if we ever change to a different tracing solution I don't think anyone will remember to come back and fix this.

Reply:
Eh, I see the concern. I'm not going to fill in the values there but I will instruct people to inject directly into context. That makes the most sense for opentracing in general.

Reply:
I don't think the idea is to let other servers implement their own opentracing and send it to us. I see this as a project to trace what's going on inside synapses. If we want it to be an optional thing that other servers implement maybe it should be spec'd but then our spec depends on opentracing's.

Reply:
Hows that?

Reply:
> I see this as a project to trace what's going on inside synapses we trust.

sure, but if we change the impl, we (or whoever is running the synapses in question) have to make sure that *all* those synapses get updated.

> Hows that?

perfect.